impress-org/givewp-test-data
# Generate Donations screen is now completed

Resolves #3

## Description

This PR completes the Generate Donations screen. Admin is now able to generate donations from Admin UI by selecting a donation status, donations count, and donation revenue.

## Affects

Generate Donations screen.

## Visuals

![deepin-screen-recorder_Select area_20201120121833](https://user-images.githubusercontent.com/4222590/99794531-95655900-2b2a-11eb-880c-3e8747d9ed0a.gif)

## Pre-review Checklist




## Testing Instructions

1. Go to Donations -> Tools -> TestData -> Generate Donations
2. Check the status of the donation which you want to generate, enter donation count and click on Generate btn
